README.md

```diff
@@ -166,9 +217,10 @@ screen is meant to be visible before your app has started, that would seem to
 defeat the purpose of the splash screen.  Providing some configuration in
 `config.xml` will automatically `show` the splash screen immediately after your
 app launch and before it has fully started and received the `deviceready`
-event. See [Icons and Splash Screens](http://cordova.apache.org/docs/en/edge/config_ref_images.md.html)
+event. See [Icons and Splash Screens][Icons and Splash Screens]
```

Comment:
Your change adds the docs here - so you need not even reference this page - correct?